client/src/App.jsx

```diff
@@ -30,72 +30,72 @@ class App extends Component {
 
     this.state = {
       isOpen: false,
-      pageID: 1,
       reviews: [],
       average: [],
-      stars: {}
-    }
+      stars: {},
```

Comment:
I don't think that coma is necessary if that's the last state on the list on line 35

Reply:
trailing comma is a thing, i don't think it's needed either but airbnb style guide lol